projects/igniteui-angular/src/lib/grids/grid/grid-cell-editing.spec.ts

```diff
@@ -694,6 +694,45 @@ describe('IgxGrid - Cell Editing #grid', () => {
             expect(cell.value).toEqual('New Name');
         });
 
+        it(`Should not update data in grid with transactions, when row is updated in onCellEdit and onCellEdit is canceled`, () => {
+            fixture = TestBed.createComponent(SelectionWithTransactionsComponent);
+            fixture.detectChanges();
+            grid = fixture.componentInstance.grid;
+            grid.ngAfterViewInit();
```

Comment:
Just noticed this one.. why'd you call after view init manually?

Reply:
nice catch. Thank you - it is removed now :)